packages/exoflex/src/components/Button/ButtonOpacity.tsx

```diff
@@ -35,47 +37,83 @@ export default function ButtonOpacity(props: ButtonProps) {
   });
   let { style: themeStyle } = useTheme();
 
+  const animatedOpacity = useRef(new Animated.Value(1)).current;
+  const fadeOpacityIn = useCallback(() => {
+    Animated.timing(animatedOpacity, {
+      toValue: preset === 'primary' ? 0.8 : 0.5,
+      duration: 100,
+      useNativeDriver: true,
+    }).start();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+  const fadeOpacityOut = useCallback(() => {
+    Animated.timing(animatedOpacity, {
+      toValue: 1,
+      duration: 150,
+      useNativeDriver: true,
+    }).start();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
   return (
-    <TouchableOpacity
+    <Pressable
       {...otherProps}
       onPress={onPress}
-      activeOpacity={preset === 'primary' ? 0.8 : 0.5}
+      onPressIn={(e) => {
+        onPressIn && onPressIn(e);
+        fadeOpacityIn();
+      }}
```

Comment:
Here, we create another arrow function inline, which is the thing we want to optimize before by moving the arrow function to useCallback.

If you want a deep dive on this, beta docs from React is a good starting point, like: https://beta.reactjs.org/reference/react/useCallback#should-you-add-usecallback-everywhere

But in the meantime, let's just keep the fix in and iterate on the optimization later.

Reply:
Got it, will do. 
For the repo setting (remove/bypass the circleci check), do you have the authority for that @darcien? I think I don't have the permission, so I can't adjust the setting